Makefile

```diff
@@ -242,7 +242,7 @@ $(BUILD_DIR):
 #######################################
 flash:
 	-./python/tastm32-dfu.py
-	sleep 1
+	sleep 5
```

Comment:
2-3 seconds should be sufficient. I know 1 wasn't always good enough. 5 probably overkill

Reply:
changed to 3.